presidio-anonymizer/presidio_anonymizer/anonymizer_engine.py

```diff
@@ -67,11 +71,21 @@ def anonymize(
             anonymized_text = self.__extract_anonymizer_and_anonymize(
                 analyzer_result.entity_type, anonymizer_config, text_to_anonymize
             )
-            text_builder.replace_text(
+            index_from_end = text_builder.replace_text(
                 anonymized_text, analyzer_result.start, analyzer_result.end
             )
 
-        return text_builder.output_text
+            result_item = AnonymizerResultItem(anonymizer_config.anonymizer_name,
+                                               analyzer_result.entity_type,
+                                               0,
+                                               index_from_end,
```

Comment:
1. Please define the parameter name (start=0 and not positional)
2. Can we already fill in the start here and skip the normalization later? isn't `start = index_from_end - len(anonymized_text)`?

Reply:
1. ok
2. no, because the while in the loop the text is being changed, and only at the end of the loop we can tell the len.